App/LoginValidate.js

```diff
@@ -0,0 +1,73 @@
+var LOGIN_SUCCESFULLY = 0;
+var INVALID_PASSWORD = 1;
+var INVALID_USERNAME = 2;
+
+var usernameJson = JSON.parse(usersJSON);
+// console.log(usernameJson);
+
+var accountService = AccountService.getInstance();
+
+document.addEventListener("DOMContentLoaded", function(event) {
+
+  for (var i = 0; i < usernameJson.length; i++) {
```

Comment:
map from json to User objects then do an AccountService.getInstance().setUsers(userModels);